<!DOCTYPE html>
<html>
<head><meta name="generator" content="Hexo 3.9.0">
  <meta charset="utf-8">
  
<!-- Google Analytics -->
<script type="text/javascript">
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-96900517-2', 'auto');
ga('send', 'pageview');

</script>
<!-- End Google Analytics -->


  
  <title>How to map NGS data using BWA-MEM | n|nja</title>
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
  <meta name="description" content="Hi sir,It is definetly autumn. I recommend to visit Miyajima and try “Momiji Manju”. Today, I introduce how to map NGS data using BWA-MEM. BWAMEM aligner is one of the most famous tools in NGS analysi">
<meta name="keywords" content="Bioinfomatics,Anaconda,SRA,Sickle,FLASh,BWA-MEM">
<meta property="og:type" content="article">
<meta property="og:title" content="How to map NGS data using BWA-MEM">
<meta property="og:url" content="https://KazukiNakamae.github.io/2019/10/27/How-to-map-NGS-data-using-BWAMEM/index.html">
<meta property="og:site_name" content="n|nja">
<meta property="og:description" content="Hi sir,It is definetly autumn. I recommend to visit Miyajima and try “Momiji Manju”. Today, I introduce how to map NGS data using BWA-MEM. BWAMEM aligner is one of the most famous tools in NGS analysi">
<meta property="og:locale" content="en">
<meta property="og:image" content="https://resources.matcha-jp.com/resize/720x2000/2019/08/05-83019.jpeg">
<meta property="og:image" content="https://kazukinakamae.github.io/images/2019-10-27_1.png">
<meta property="og:image" content="https://kazukinakamae.github.io/images/2019-10-27_2.png">
<meta property="og:image" content="https://kazukinakamae.github.io/images/2019-11-7_4.png">
<meta property="og:updated_time" content="2019-11-06T18:03:37.802Z">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="How to map NGS data using BWA-MEM">
<meta name="twitter:description" content="Hi sir,It is definetly autumn. I recommend to visit Miyajima and try “Momiji Manju”. Today, I introduce how to map NGS data using BWA-MEM. BWAMEM aligner is one of the most famous tools in NGS analysi">
<meta name="twitter:image" content="https://resources.matcha-jp.com/resize/720x2000/2019/08/05-83019.jpeg">
<meta name="twitter:creator" content="@kazuki_nakamae">
  
    <link rel="alternate" href="/atom.xml" title="n|nja" type="application/atom+xml">
  
  
    <link rel="icon" href="/favicon.png">
  
  
    <link href="//fonts.googleapis.com/css?family=Source+Code+Pro" rel="stylesheet" type="text/css">
  
  <link rel="stylesheet" href="/css/style.css">
</head>
</html>
<body>
  <div id="container">
    <div id="wrap">
      <header id="header">
  <div id="banner"></div>
  <div id="header-outer" class="outer">
    <div id="header-title" class="inner">
      <h1 id="logo-wrap">
        <a href="/" id="logo">n|nja</a>
      </h1>
      
        <h2 id="subtitle-wrap">
          <a href="/" id="subtitle">EYE FROM SUNRISE</a>
        </h2>
      
    </div>
    <div id="header-inner" class="inner">
      <nav id="main-nav">
        <a id="main-nav-toggle" class="nav-icon"></a>
        
          <a class="main-nav-link" href="/">Home</a>
        
          <a class="main-nav-link" href="/categories">Categories</a>
        
          <a class="main-nav-link" href="/tags">Tags</a>
        
          <a class="main-nav-link" href="/archives">Archives</a>
        
      </nav>
      <nav id="sub-nav">
        
          <a id="nav-rss-link" class="nav-icon" href="/atom.xml" title="RSS Feed"></a>
        
        <a id="nav-search-btn" class="nav-icon" title="Search"></a>
      </nav>
      <div id="search-form-wrap">
        <form action="//google.com/search" method="get" accept-charset="UTF-8" class="search-form"><input type="search" name="q" class="search-form-input" placeholder="Search"><button type="submit" class="search-form-submit">&#xF002;</button><input type="hidden" name="sitesearch" value="https://KazukiNakamae.github.io"></form>
      </div>
    </div>
  </div>
</header>
      <div class="outer">
        <section id="main"><article id="post-How-to-map-NGS-data-using-BWAMEM" class="article article-type-post" itemscope itemprop="blogPost">
  <div class="article-meta">
    <a href="/2019/10/27/How-to-map-NGS-data-using-BWAMEM/" class="article-date">
  <time datetime="2019-10-27T06:24:22.000Z" itemprop="datePublished">2019-10-27</time>
</a>
    
  <div class="article-category">
    <a class="article-category-link" href="/categories/NGS-Analysis/">NGS Analysis</a>
  </div>

  </div>
  <div class="article-inner">
    
    
      <header class="article-header">
        
  
    <h1 class="article-title" itemprop="name">
      How to map NGS data using BWA-MEM
    </h1>
  

      </header>
    
    <div class="article-entry" itemprop="articleBody">
      
        <p>Hi sir,<br>It is definetly autumn. I recommend to visit <a href="https://matcha-jp.com/en/5124" title="Miyajima" target="_blank" rel="noopener">Miyajima</a> and try “Momiji Manju”.<br><img src="https://resources.matcha-jp.com/resize/720x2000/2019/08/05-83019.jpeg" alt="Miyajima (cited from MATCHA)" title="Cited from MATCHA"></p>
<p>Today, I introduce how to map NGS data using BWA-MEM.</p>
<p><a href="http://bio-bwa.sourceforge.net" title="BWAMEM aligner" target="_blank" rel="noopener">BWAMEM aligner</a> is one of the most famous tools in NGS analysis. Some tool requires data processed using BWA-MEM (It is difficult for beginners…).</p>
<p>I would like to suggest that you buy &gt;20GB external hard drive/SSD/USB flash drive before you try the following analysis because NGS analysis manages numerous amounts of data. I usually 500 GB external hard drive to save NGS-associated data (saving directory: <code>/Volumes/databank1/ngs</code>). I don’t use my internal storage, it’s for Steam.</p>
<p>We introduce it, step by step. If you want to see the only mapping step, scroll down to “STEP4”.</p>
<h3 id="STEP1-Creating-a-visual-environment"><a href="#STEP1-Creating-a-visual-environment" class="headerlink" title="STEP1: Creating a visual environment"></a>STEP1: Creating a visual environment</h3><h4 id="1-Create-visual-environment"><a href="#1-Create-visual-environment" class="headerlink" title="1: Create visual environment"></a>1: Create visual environment</h4><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">conda create --name bwa_env;</span><br></pre></td></tr></table></figure>

<p>*If you cannot use <code>conda</code> command in your terminal, please visit <a href="https://kazukinakamae.github.io/2019/10/26/How-to-install-Anaconda/" title="How to install Anaconda">How to install Anaconda</a>.</p>
<h4 id="2-Activate-the-visual-environment"><a href="#2-Activate-the-visual-environment" class="headerlink" title="2: Activate the visual environment"></a>2: Activate the visual environment</h4><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line"><span class="built_in">source</span> activate bwa_env;</span><br></pre></td></tr></table></figure>

<h4 id="3-Add-packages-into-the-visual-environment"><a href="#3-Add-packages-into-the-visual-environment" class="headerlink" title="3: Add packages into the visual environment"></a>3: Add packages into the visual environment</h4><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">conda config --add channels bioconda; <span class="comment"># add channel</span></span><br><span class="line">conda install --channel bioconda sra-tools sickle-trim flash bwa samtools; <span class="comment"># install packages from the channel</span></span><br><span class="line">conda config --remove channels bioconda; <span class="comment"># remove the channel</span></span><br></pre></td></tr></table></figure>

<h3 id="STEP2-Preparing-FASTQ-file"><a href="#STEP2-Preparing-FASTQ-file" class="headerlink" title="STEP2: Preparing FASTQ file"></a>STEP2: Preparing FASTQ file</h3><p>*In this example, I use published amplicon sequence data. If you have the filtered &amp; merged FASTQ files, skip this section.</p>
<h4 id="4-Download-compressed-FASTQ-files"><a href="#4-Download-compressed-FASTQ-files" class="headerlink" title="4: Download compressed FASTQ files"></a>4: Download compressed FASTQ files</h4><p><a href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=DRR147084" title="DRR147084" target="_blank" rel="noopener">Information Page of DRR147084 in SRA (DRR database)</a><br>You don’t have to access the site when you download it. Type the following command.</p>
<figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">fastq-dump DRR147084 --split-files --gzip --outdir /Volumes/databank1/ngs; <span class="comment"># Download from SRR(DRR)</span></span><br></pre></td></tr></table></figure>

<h4 id="5-Wait-for-a-minute-until-you-can-type-a-command-in-your-terminal-again"><a href="#5-Wait-for-a-minute-until-you-can-type-a-command-in-your-terminal-again" class="headerlink" title="5: Wait for a minute until you can type a command in your terminal again"></a>5: Wait for a minute until you can type a command in your terminal again</h4><p>*You can kill time by reading <a href="https://www.nature.com/articles/s41467-018-05773-6" title="paper" target="_blank" rel="noopener">this paper</a> which is the original report using the downloading deep sequencing data.</p>
<h4 id="6-Filter-the-raw-NGS-files-using-sickle"><a href="#6-Filter-the-raw-NGS-files-using-sickle" class="headerlink" title="6: Filter the raw NGS files using sickle"></a>6: Filter the raw NGS files using sickle</h4><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">sickle pe -f /Volumes/databank1/ngs/DRR147084_1.fastq.gz -r /Volumes/databank1/ngs/DRR147084_2.fastq.gz -t sanger \</span><br><span class="line">-o /Volumes/databank1/ngs/qfltr_DRR147084_1.fastq -p /Volumes/databank1/ngs/qfltr_DRR147084_2.fastq \</span><br><span class="line">-s /Volumes/databank1/ngs/single_DRR147084.fastq; <span class="comment"># Trim average quality &lt;\Q20 and read length &lt; 20bp</span></span><br></pre></td></tr></table></figure>

<p>*<code>qfltr_DRR147084_1.fastq</code>, <code>qfltr_DRR147084_2.fastq</code>, <code>single_DRR147084.fastq</code> are my choice. You can name it freely.<br>*Do you want to trim adapters before that? You are such a nice guy!!! You can check <a href="https://kazukinakamae.github.io/2019/10/28/How-to-trim-adapters-using-Trimmomatic/" title="How to trim adapters using Trimmomatic">the article</a>.</p>
<h4 id="7-Merge-paired-end-reads-using-FLASh"><a href="#7-Merge-paired-end-reads-using-FLASh" class="headerlink" title="7: Merge paired-end reads using FLASh"></a>7: Merge paired-end reads using FLASh</h4><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line"><span class="built_in">cd</span> /Volumes/databank1/ngs/;flash /Volumes/databank1/ngs/qfltr_DRR147084_1.fastq /Volumes/databank1/ngs/qfltr_DRR147084_2.fastq;<span class="built_in">cd</span> ~/Desktop;</span><br></pre></td></tr></table></figure>

<p>The merged file “out.extendedFrags.fastq” is saved into <code>/Volumes/databank1/ngs/</code> when the process is done.</p>
<h3 id="STEP3-Generate-the-fasta-file-index-of-the-human-reference-genome-version-38"><a href="#STEP3-Generate-the-fasta-file-index-of-the-human-reference-genome-version-38" class="headerlink" title="STEP3: Generate the fasta file index of the human reference genome (version 38)"></a>STEP3: Generate the fasta file index of the human reference genome (version 38)</h3><p>*If you got the NGS data from the other organism, you have to try it using reference genome of that. <a href="http://hgdownload.cse.ucsc.edu/downloads.html" title="UCSC" target="_blank" rel="noopener">This site</a> will be helpful to find the genome you want.</p>
<h4 id="8-Download-compressed-files-of-the-human-reference-genome-version-38"><a href="#8-Download-compressed-files-of-the-human-reference-genome-version-38" class="headerlink" title="8: Download compressed files of the human reference genome (version 38)"></a>8: Download compressed files of the human reference genome (version 38)</h4><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">wget http://hgdownload.cse.ucsc.edu/goldenPath/hg38/bigZips/analysisSet/hg38.analysisSet.chroms.tar.gz --directory-prefix /Volumes/databank1/ngs/;</span><br></pre></td></tr></table></figure>

<h4 id="9-Unpack-the-files"><a href="#9-Unpack-the-files" class="headerlink" title="9: Unpack the files"></a>9: Unpack the files</h4><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line"><span class="built_in">cd</span> /Volumes/databank1/ngs/;</span><br><span class="line">gunzip -c /Volumes/databank1/ngs/hg38.analysisSet.chroms.tar.gz | tar xopf -;</span><br><span class="line"><span class="built_in">cd</span> ~/Desktop;</span><br></pre></td></tr></table></figure>

<h4 id="10-Conbine-files-into-hg38-fa"><a href="#10-Conbine-files-into-hg38-fa" class="headerlink" title="10: Conbine files into hg38.fa"></a>10: Conbine files into hg38.fa</h4><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line"><span class="keyword">for</span> i <span class="keyword">in</span> `seq 1 22` X Y;<span class="keyword">do</span> cat /Volumes/databank1/ngs/hg38.analysisSet.chroms/chr<span class="variable">$i</span>.fa &gt;&gt; /Volumes/databank1/ngs/hg38.fa;<span class="keyword">done</span>;</span><br></pre></td></tr></table></figure>

<h4 id="11-Delete-immediate-files-to-save-storage"><a href="#11-Delete-immediate-files-to-save-storage" class="headerlink" title="11: Delete immediate files to save storage"></a>11: Delete immediate files to save storage</h4><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">rm -rf /Volumes/databank1/ngs/hg38.analysisSet.chroms/;</span><br></pre></td></tr></table></figure>

<h4 id="12-Generate-the-fasta-file-index-of-hg38"><a href="#12-Generate-the-fasta-file-index-of-hg38" class="headerlink" title="12: Generate the fasta file index of hg38"></a>12: Generate the fasta file index of hg38</h4><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">bwa index -p /Volumes/databank1/ngs/hg38 /Volumes/databank1/ngs/hg38.fa;</span><br></pre></td></tr></table></figure>

<h4 id="13-Wait-for-about-20-30-minutes-until-you-can-type-a-command-in-your-terminal-again"><a href="#13-Wait-for-about-20-30-minutes-until-you-can-type-a-command-in-your-terminal-again" class="headerlink" title="13: Wait for about 20-30 minutes until you can type a command in your terminal again"></a>13: Wait for about 20-30 minutes until you can type a command in your terminal again</h4><p>You can take a nap and browse the shops. I like a squid jerky of 7-Eleven…</p>
<h3 id="STEP4-Mapping-the-merged-FASTQ-file-using-human-reference-genome-hg38"><a href="#STEP4-Mapping-the-merged-FASTQ-file-using-human-reference-genome-hg38" class="headerlink" title="STEP4: Mapping the merged FASTQ file using human reference genome (hg38)"></a>STEP4: Mapping the merged FASTQ file using human reference genome (hg38)</h3><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">bwa mem -t 1 /Volumes/databank1/ngs/hg38 /Volumes/databank1/ngs/out.extendedFrags.fastq | samtools sort -o /Volumes/databank1/ngs/out.extendedFrags.fastq.bam;</span><br></pre></td></tr></table></figure>

<p>*<code>/Volumes/databank1/ngs/hg38</code> indicates the name of the reference index.</p>
<p>The mapped bam file “out.extendedFrags.fastq.bam” is saved into <code>/Volumes/databank1/ngs/</code> when the process is done.<br>I guess that you can use it for some analysis.</p>
<h3 id="Option-View-the-mapped-bam-file-using-Integrative-Genomics-Viewer-IGV"><a href="#Option-View-the-mapped-bam-file-using-Integrative-Genomics-Viewer-IGV" class="headerlink" title="(Option): View the mapped bam file using Integrative Genomics Viewer (IGV)"></a>(Option): View the mapped bam file using Integrative Genomics Viewer (IGV)</h3><h4 id="14-make-index-of-mapped-file"><a href="#14-make-index-of-mapped-file" class="headerlink" title="14: make index of mapped file."></a>14: make index of mapped file.</h4><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">samtools index /Volumes/databank1/ngs/out.extendedFrags.fastq.bam;</span><br></pre></td></tr></table></figure>

<h4 id="15-Download-IGV"><a href="#15-Download-IGV" class="headerlink" title="15: Download IGV"></a>15: Download IGV</h4><p><a href="https://software.broadinstitute.org/software/igv/download" title="Download Page" target="_blank" rel="noopener">Download Page</a></p>
<p><img src="/images/2019-10-27_1.png" alt="Download Page For IGV"></p>
<h4 id="16-Install-application"><a href="#16-Install-application" class="headerlink" title="16: Install application"></a>16: Install application</h4><p>When I finished the download, the application “IGV_2.7.2.app” has been already installed in the Download directory.</p>
<h4 id="17-Start-IGV-2-7-2-app"><a href="#17-Start-IGV-2-7-2-app" class="headerlink" title="17: Start IGV_2.7.2.app"></a>17: Start IGV_2.7.2.app</h4><h4 id="18-Click-“Genomes”-gt-”Load-Genome-File…”"><a href="#18-Click-“Genomes”-gt-”Load-Genome-File…”" class="headerlink" title="18: Click “Genomes”&gt;”Load Genome File…”"></a>18: Click “Genomes”&gt;”Load Genome File…”</h4><h4 id="19-Select-“-Volumes-databank1-ngs-hg38-fa”"><a href="#19-Select-“-Volumes-databank1-ngs-hg38-fa”" class="headerlink" title="19: Select “/Volumes/databank1/ngs/hg38.fa”"></a>19: Select “/Volumes/databank1/ngs/hg38.fa”</h4><h4 id="20-Click-“File”-gt-“Load-from-File…”"><a href="#20-Click-“File”-gt-“Load-from-File…”" class="headerlink" title="20: Click “File” &gt; “Load from File…”"></a>20: Click “File” &gt; “Load from File…”</h4><h4 id="21-Select-Volumes-databank1-ngs-out-extendedFrags-fastq-bam"><a href="#21-Select-Volumes-databank1-ngs-out-extendedFrags-fastq-bam" class="headerlink" title="21: Select /Volumes/databank1/ngs/out.extendedFrags.fastq.bam"></a>21: Select /Volumes/databank1/ngs/out.extendedFrags.fastq.bam</h4><h4 id="22-Enter-the-genome-region-of-the-amplicon-sequence-in-the-search-form"><a href="#22-Enter-the-genome-region-of-the-amplicon-sequence-in-the-search-form" class="headerlink" title="22: Enter the genome region of the amplicon sequence in the search form"></a>22: Enter the genome region of the amplicon sequence in the search form</h4><p>*The “DRR147084” is amplicon sequence data of ATP5B MMEJ-assisted knock-in sample. The gene region of ATP5B is “chr12:56,638,302-56,638,353”</p>
<p><img src="/images/2019-10-27_2.png" alt="amplicon sequence data in IGV"></p>
<p>*You can check the region using <a href="https://genome-asia.ucsc.edu/cgi-bin/hgTracks?db=hg38&lastVirtModeType=default&lastVirtModeExtraState=&virtModeType=default&virtMode=0&nonVirtPosition=&position=chr12%3A56638175%2D56645279&hgsid=729668905_tDpmuTzz6AxKqxPykn5I4ebXPh6r" title="genome browser" target="_blank" rel="noopener">genome browser</a>.</p>
<p>This is a simple example. You can change the procedure because there is no royal road to NGS analysis (fortunately or unfortunately, I don’t know…).</p>
<p>Additionally, you can get the following variant calling if you use specific tools such as <a href="https://bioconductor.org/packages/release/bioc/vignettes/CrispRVariants/inst/doc/user_guide.pdf" title="CrispRVariants" target="_blank" rel="noopener">CrispRVariants</a>.</p>
<p><img src="/images/2019-11-7_4.png" alt="amplicon sequence data in CrispRVariants"></p>
<h5 id="Environement"><a href="#Environement" class="headerlink" title="Environement"></a>Environement</h5><p>Hardware</p>
<ul>
<li>MacBook Air (2017 model)</li>
<li>Processor 1.8 GHz Intel Core i5</li>
<li>Memory 8GB</li>
<li>500 GB external hard drive</li>
</ul>
<p>Software</p>
<ul>
<li>macOS Mojave</li>
<li>GNU bash version : 3.2.57(1)-release (x86_64-apple-darwin18)</li>
<li>conda version : 4.7.11</li>
<li>conda-build version : 3.17.6</li>
<li>python version : 2.7.15.final.0</li>
</ul>
<p>Thank you so much…</p>

      
    </div>
    <footer class="article-footer">
      <a data-url="https://KazukiNakamae.github.io/2019/10/27/How-to-map-NGS-data-using-BWAMEM/" data-id="ck2nlb3tb00006yv11tsr4ehf" class="article-share-link">Share</a>
      
        <a href="https://KazukiNakamae.github.io/2019/10/27/How-to-map-NGS-data-using-BWAMEM/#disqus_thread" class="article-comment-link">Comments</a>
      
      
  <ul class="article-tag-list"><li class="article-tag-list-item"><a class="article-tag-list-link" href="/tags/Anaconda/">Anaconda</a></li><li class="article-tag-list-item"><a class="article-tag-list-link" href="/tags/BWA-MEM/">BWA-MEM</a></li><li class="article-tag-list-item"><a class="article-tag-list-link" href="/tags/Bioinfomatics/">Bioinfomatics</a></li><li class="article-tag-list-item"><a class="article-tag-list-link" href="/tags/FLASh/">FLASh</a></li><li class="article-tag-list-item"><a class="article-tag-list-link" href="/tags/SRA/">SRA</a></li><li class="article-tag-list-item"><a class="article-tag-list-link" href="/tags/Sickle/">Sickle</a></li></ul>

    </footer>
  </div>
  
    
<nav id="article-nav">
  
    <a href="/2019/10/28/How-to-trim-adapters-using-Trimmomatic/" id="article-nav-newer" class="article-nav-link-wrap">
      <strong class="article-nav-caption">Newer</strong>
      <div class="article-nav-title">
        
          How to trim adapters using Trimmomatic
        
      </div>
    </a>
  
  
    <a href="/2019/10/26/How-to-install-Anaconda/" id="article-nav-older" class="article-nav-link-wrap">
      <strong class="article-nav-caption">Older</strong>
      <div class="article-nav-title">How to install Anaconda</div>
    </a>
  
</nav>

  
</article>


<section id="comments">
  <div id="disqus_thread">
    <noscript>Please enable JavaScript to view the <a href="//disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
  </div>
</section>


  <div id="disqus_thread">
      <noscript>Please enable JavaScript to view the <a href="//disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
  </div>
</section>
        
          <aside id="sidebar">
  
    
  <div class="widget-wrap">
    <h3 class="widget-title">Categories</h3>
    <div class="widget">
      <ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/Installation/">Installation</a></li><li class="category-list-item"><a class="category-list-link" href="/categories/NGS-Analysis/">NGS Analysis</a></li><li class="category-list-item"><a class="category-list-link" href="/categories/others/">others</a></li></ul>
    </div>
  </div>


  
    
  <div class="widget-wrap">
    <h3 class="widget-title">Tags</h3>
    <div class="widget">
      <ul class="tag-list"><li class="tag-list-item"><a class="tag-list-link" href="/tags/Anaconda/">Anaconda</a></li><li class="tag-list-item"><a class="tag-list-link" href="/tags/BWA-MEM/">BWA-MEM</a></li><li class="tag-list-item"><a class="tag-list-link" href="/tags/Bioinfomatics/">Bioinfomatics</a></li><li class="tag-list-item"><a class="tag-list-link" href="/tags/CrispRVariants/">CrispRVariants</a></li><li class="tag-list-item"><a class="tag-list-link" href="/tags/FLASh/">FLASh</a></li><li class="tag-list-item"><a class="tag-list-link" href="/tags/RStudio/">RStudio</a></li><li class="tag-list-item"><a class="tag-list-link" href="/tags/SRA/">SRA</a></li><li class="tag-list-item"><a class="tag-list-link" href="/tags/Sickle/">Sickle</a></li><li class="tag-list-item"><a class="tag-list-link" href="/tags/Trimmomatic/">Trimmomatic</a></li><li class="tag-list-item"><a class="tag-list-link" href="/tags/report/">report</a></li></ul>
    </div>
  </div>


  
    
  <div class="widget-wrap">
    <h3 class="widget-title">Tag Cloud</h3>
    <div class="widget tagcloud">
      <a href="/tags/Anaconda/" style="font-size: 16.67px;">Anaconda</a> <a href="/tags/BWA-MEM/" style="font-size: 10px;">BWA-MEM</a> <a href="/tags/Bioinfomatics/" style="font-size: 20px;">Bioinfomatics</a> <a href="/tags/CrispRVariants/" style="font-size: 10px;">CrispRVariants</a> <a href="/tags/FLASh/" style="font-size: 10px;">FLASh</a> <a href="/tags/RStudio/" style="font-size: 10px;">RStudio</a> <a href="/tags/SRA/" style="font-size: 13.33px;">SRA</a> <a href="/tags/Sickle/" style="font-size: 10px;">Sickle</a> <a href="/tags/Trimmomatic/" style="font-size: 10px;">Trimmomatic</a> <a href="/tags/report/" style="font-size: 10px;">report</a>
    </div>
  </div>

  
    
  <div class="widget-wrap">
    <h3 class="widget-title">Archives</h3>
    <div class="widget">
      <ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/11/">November 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/10/">October 2019</a></li></ul>
    </div>
  </div>


  
    
  <div class="widget-wrap">
    <h3 class="widget-title">Recent Posts</h3>
    <div class="widget">
      <ul>
        
          <li>
            <a href="/2019/11/07/how-to-map-CRISPR-Cas9-sample-CrispRVariants/">How to analyze CRISPR-Cas9 sample using CrispRVariants</a>
          </li>
        
          <li>
            <a href="/2019/10/28/How-to-trim-adapters-using-Trimmomatic/">How to trim adapters using Trimmomatic</a>
          </li>
        
          <li>
            <a href="/2019/10/27/How-to-map-NGS-data-using-BWAMEM/">How to map NGS data using BWA-MEM</a>
          </li>
        
          <li>
            <a href="/2019/10/26/How-to-install-Anaconda/">How to install Anaconda</a>
          </li>
        
          <li>
            <a href="/2019/10/25/Launched/">Launched</a>
          </li>
        
      </ul>
    </div>
  </div>

  
</aside>
        
      </div>
      <footer id="footer">
  
  <div class="outer">

    
      <script type="text/javascript">
          var disqus_shortname = 'kazukinakamae-com';
          var disqus_config = function () {
              this.page.url = 'https://KazukiNakamae.github.io/2019/11/07/how-to-map-CRISPR-Cas9-sample-CrispRVariants/';
              this.page.identifier = '/2019/11/07/how-to-map-CRISPR-Cas9-sample-CrispRVariants/';
              this.language = 'en';
          };
          (function(){
              var dsq = document.createElement('script');
              dsq.type = 'text/javascript';
              dsq.async = true;
              dsq.src =  '//a.disquscdn.com/embed.js';
              (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
          }());
      </script>
    

    <div id="footer-info" class="inner">
      &copy; 2019 Kazuki Nakamae<br>
      Powered by <a href="http://hexo.io/" target="_blank">Hexo</a>
    </div>
  </div>
</footer>
    </div>
    <nav id="mobile-nav">
  
    <a href="/" class="mobile-nav-link">Home</a>
  
    <a href="/categories" class="mobile-nav-link">Categories</a>
  
    <a href="/tags" class="mobile-nav-link">Tags</a>
  
    <a href="/archives" class="mobile-nav-link">Archives</a>
  
</nav>
    
<script>
  var disqus_shortname = 'kazukinakamae-com';
  
  var disqus_url = 'https://KazukiNakamae.github.io/2019/10/27/How-to-map-NGS-data-using-BWAMEM/';
  
  (function(){
    var dsq = document.createElement('script');
    dsq.type = 'text/javascript';
    dsq.async = true;
    dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
    (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
  })();
</script>


<script src="//ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>


  <link rel="stylesheet" href="/fancybox/jquery.fancybox.css">
  <script src="/fancybox/jquery.fancybox.pack.js"></script>


<script src="/js/script.js"></script>



  </div>
</body>
</html>